front-end/src/components/layout/Breadcrumbs.js

```diff
@@ -0,0 +1,57 @@
+import { useLocation } from "react-router-dom";
+import MUIBreadcrumbs from "@mui/material/Breadcrumbs";
+import Paper from "@mui/material/Paper";
+import Container from "@mui/system/Container";
+
+import Link from "./Link";
+
+import "./Breadcrumbs.css";
+
+function getPathFriendlyName(path) {
+  const friendlyNames = {
+    register: "Register",
+    login: "Login",
+    find: "Find Tables",
+    "view-table/d+": "View Table",
+    "submit-table": "Submit Table",
+  };
+
+  const entries = Object.entries(friendlyNames);
+  const lookup = entries.map(([regex, name]) => ({ regex, name }));
+  const entry = lookup.find(({ regex }) => path.match(regex));
+
+  return entry?.name;
+}
+
+function createPathBreadcrumbLink(_, index, paths) {
+  const urlPath = paths.slice(0, index + 1).join("/");
+  const friendlyName = getPathFriendlyName(urlPath);
+
+  if (!friendlyName) {
+    return undefined;
+  }
+
+  return (
+    <Link underline="hover" to={urlPath} key={urlPath}>
+      {friendlyName}
+    </Link>
+  );
+}
+
+export default function Breadcrumbs() {
+  const location = useLocation();
+  const paths = location.pathname.split("/").filter((x) => !!x);
+
+  return (
+    <Container maxWidth="md">
+      <Paper className="breadcrumbs-wrapper">
+        <MUIBreadcrumbs aria-label="breadcrumbs">
+          <Link underline="hover" to="/">
+            Home
+          </Link>
+          {paths.map(createPathBreadcrumbLink)}

```

Comment:
Another small thing - do you think it makes sense to make the current page link stand out in some way? possibly make it bold? and maybe make it unclickable too?

Reply:
I originally had some logic that handled the current link differently (used the `Typography` component, like [this demo in the MUI docs](https://mui.com/material-ui/react-breadcrumbs/#integration-with-react-router)

So yep, I can definitely add that back! Going to create a separate issue for this to keep this review small. IMO it's a lot easier and faster for everyone to work on and review smaller incremental changes rather than lumping it all into one issue.

Reply:
Good call on that!

Reply:
I combined this with the previous change, so this is also tracked in #84